Sources/LiveViewNative/Modifiers/Input Events Modifiers/DigitalCrownAccessoryModifier.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  DigitalCrownAccessoryModifier.swift
+//  LiveViewNative
+//
+//  Created by Dylan.Ginsburg on 4/26/23.
+//
+
+import SwiftUI
+
+/// Places an accessory View next to the Digital Crown on Apple Watch.
+///
+/// Nested content is referenced by its namespace using the `content` argument.
+///
+/// ```html
+/// <List modifiers={@native |> digital_crown_accessory(content: :dca_view)}>
+///     ...
+///     <digital_crown_accessory:dca_view>
+///         <Image system-name="heart.fill" modifiers={@native |> digital_crown_accessory_visibility(visibility: :visible)} />
+///     </digital_crown_accessory:dca_view>
+/// </List>
+/// ```
```

Comment:
Not feeling really confident about this example and I'm not sure how to test it.

Reply:
This example mostly looks good, but the `digital_crown_accessory_visibility` modifier needs to go around the `digital_crown_accessory` modifier. The use case for this is fairly narrow, I think it'd be more useful if we supported `digitalCrownRotation`.

Reply:
`digitalCrownRotation` was put in the rejected column a few weeks ago. #601

@carson-katri @AZholtkevych Do we want to add it back?